src/reference-manual/analysis.Rmd

```diff
@@ -6,12 +6,21 @@ Markov chain Monte Carlo (MCMC) methods were developed for situations
 in which it is not straightforward to make independent draws
 @Metropolis:1953.
 
-In addition to providing point estimates, Stan's optimization algorithm
-provides a Laplace approximation from which it is easy to draw
-random values.  Stan's variational inference algorithm provides
-draws from the variational approximation to the posterior.  Both
-of these outputs may be analyzed just as any other MCMC output,
-despite the fact that it is actually independent draws.
+Stan's variational inference algorithm provides draws from the variational
+approximation to the posterior which may be analyzed just as any other MCMC output,
+despite the fact that it is not actually a Markov chain.
+
+Stan's Laplace algorithm
+produces a samples from a normal approximation
+centered at the mode of a distribution.
+Stan's Laplace algorithm produces a sample from a normal approximation
+centered at the mode of a distribution in the unconstrained space.
```

Comment:
remove the extra lines

Reply:
good catch - fixed!
but that is really weird - I think git somehow merged old/new versions.